src/test/java/blackjack/domain/ResultTest.java

```diff
@@ -0,0 +1,35 @@
+package blackjack.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class ResultTest {
+
+    @DisplayName("결과를 생성한다")
+    @Test
+    public void create() {
+        assertThatCode(() -> new Result(Map.of(new Player("mark"), ResultStatus.WIN)))
+                .doesNotThrowAnyException();
+    }
+
+    @DisplayName("딜러의 승패를 반환한다")
+    @Test
+    public void dealerResult() {
+        Result result = new Result(
+                Map.of(new Player("mark"), ResultStatus.DRAW, new Player("isang"), ResultStatus.WIN,
+                        new Player("isang"), ResultStatus.WIN));
```

Comment:
```suggestion
        Result result = new Result(
                Map.of(
                    new Player("mark"), ResultStatus.DRAW,
                    new Player("isang"), ResultStatus.WIN,
                    new Player("isang"), ResultStatus.WIN));
```
리스트, 맵과 같은 컬렉션에 값을 채워넣을 때 가독성을 높이는 방식으로 그 값들을 나열해봐도 좋을 것 같네요!

Reply:
테스트라고 계속 무신경하게 넘어가나 봅니다. 테스트도 유지보수 대상의 코드라고 생각하며 집중할게요!

Reply:
저는 테스트 유지보수가 프로덕션 코드 유지보수보다 훨씬 어렵다고 생각해요. 그래서 테스트에 꽤 배울점이나 대화할 요소가 많은 것 같네요!